app/.eslintrc.json

```diff
@@ -1,37 +1,83 @@
 {
-  "extends": ["airbnb-base", "airbnb-typescript/base"],
-  "parserOptions": {
-    "project": "./tsconfig.json"
-  },
-  "rules": {
-    "import/prefer-default-export": "off",
-    "@typescript-eslint/no-use-before-define": "off",
-    "@typescript-eslint/no-useless-constructor": "off",
-    "no-param-reassign": [2, { "props": false }],
-    "no-console": "off",
-    "no-return-await": "off",
-    "no-confusing-arrow": [
-      "warn",
-      { "allowParens": true }
-    ],
-    "implicit-arrow-linebreak": 0,
-	"arrow-body-style": 0,
-	"max-len": [
-	  "error",
-	  {
-		  "code": 120,
-		  "ignoreStrings": true
-	  }
+	"extends": [
+		"airbnb-base",
```

Comment:
If anyone was curious about what this is: https://www.npmjs.com/package/eslint-config-airbnb-base